lesson8/layout/index.html

```diff
@@ -0,0 +1,343 @@
+                        <div class="col-xl-5 col-lg-5 col-md-5 ">
+                            <div class="header__content__wrapper">
+                                <div class="row">
+                                    <div
+                                        class="col-lg-12 col-sm-6 col-6 padding__right--clear padding__xs__right padding__xs__right--clear">
+                                        <div class="header__content__advertisement background--blue">
+                                            <div class="header__content__advertisement__body">
+                                                <div class="header__content__advertiserment__heading">
+                                                    Sed lorem
+                                                </div>
+
+                                                <div class="header__content__advertiserment__text">
+                                                    Sed egenstas, ante vulputate volutpat vitae luctus metus libero
+                                                    aliquam
+                                                </div>
+
+                                                <button
+                                                    class="btn btn__site header__content__advertiserment__btn  btn--blue">More</button>
+                                            </div>
+                                        </div>
+                                    </div>
+
+                                    <div
+                                        class="col-lg-12 col-sm-6 col-6 padding__right--clear padding__xs__right padding__xs__left--clear">
+                                        <div class="header__content__advertisement background--cyan">
+                                            <div class="header__content__advertisement__body">
+                                                <div class="header__content__advertiserment__heading">
+                                                    AMET MAGNA
+                                                </div>
+
+                                                <div class="header__content__advertiserment__text">
+                                                    Fusce lancinia ipsum feugiat dolor sed matis tempus sed nullam
+                                                </div>
+
+                                                <button
+                                                    class="btn btn__site header__content__advertiserment__btn  btn--cyan">More
+                                                </button>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-xl-7 col-lg-7 col-md-7 col-12 padding__left--clear padding__xs__left">

```

Comment:
```suggestion
                        </div>
                   
                        <div class="col-xl-7 col-lg-7 col-md-7 col-12 padding__left--clear padding__xs__left">

```
- Tách nhau ra
- `padding__left--clear padding__xs__left` viết ở chỗ khác, không viết ở đây